src/main/java/com/redhat/devtools/lsp4ij/ui/IconMapper.java

```diff
@@ -242,6 +243,41 @@ private static boolean isValidHexColor(@Nullable String hexValue) {
         }
     }
 
+    /**
+     * Returns the icon for the provided {@link SemanticTokenTypes} value if possible.
+     *
+     * @param tokenType the icon name as semantic token type value
+     * @return the corresponding icon, or null if no icon could be found
+     */
+    @Nullable
+    public static Icon getIcon(@Nullable String tokenType) {
```

Comment:
Added another signature to map a value from `SemanticTokenTypes` to the appropriate icon.